### PR TITLE
Update existential deposits

### PR DESCRIPTION
**Pull request by laboon**
They are different on Kusama  and Polkadot

### CODE REVIEW

File: docs/learn-accounts.md [hunk 1]
```diff
@@ -100,14 +100,14 @@ Hardware wallet integration from [Ledger](https://ledger.com) is coming soon.
...
-state to conserve space. You do not lose access to the reaped address - as long as you have your
-private key or recovery phrase, you can still use the address - but it needs a top-up of another
-existential deposit to be able to interact with the chain.
+account does not exist yet on-chain. For that, it needs the existential deposit - 0.001 KSM (on
+Kusama) or 0.01 DOT (old) / 1 _New DOT_ (on Polkadot mainnet) .
+
+Having an account go below the existential deposit causes that account to be _reaped_. The account
+will be wiped from the blockchain's state to conserve space, along with any funds in that address.
+You do not lose access to the reaped address - as long as you have your private key or recovery
+phrase, you can still use the address - but it needs a top-up of another existential deposit to be
```
Comment (krichard410) [inline docs/learn-accounts.md:109]: I'm just wondering how would you know if it's top-up of another existential deposit?

Comment (lsaether) [reply docs/learn-accounts.md:109]: You just won't be able to use the account until you top it up again and the balance will read 0 DOT.

File: docs/learn-accounts.md [hunk 2]
```diff
@@ -100,14 +100,14 @@ Hardware wallet integration from [Ledger](https://ledger.com) is coming soon.
...
 When you generate an account (address), you only generate a _key_ that lets you access it. The
-account does not exist yet on-chain. For that, it needs the existential deposit - 0.01 KSM / DOT.
-This deposit is paid by the account sending you tokens. The one "creating" you pays for your
-creation.
-
-Going below 0.01 causes an account to be _reaped_. Your account will be wiped from the blockchain's
-state to conserve space. You do not lose access to the reaped address - as long as you have your
-private key or recovery phrase, you can still use the address - but it needs a top-up of another
-existential deposit to be able to interact with the chain.
+account does not exist yet on-chain. For that, it needs the existential deposit - 0.001 KSM (on
```
Comment (krichard410) [inline docs/learn-accounts.md]:
```suggestion
account does not exist yet on-chain. For that, it needs the existential deposit: 0.001 KSM (on
```
Could we change the - to a : instead? It looked like the number was negative

Comment (Swader) [reply docs/learn-accounts.md]:
```bash
const balances.existentialDeposit: Balance
1.666m KSM
```

**Review by Swader — CHANGES_REQUESTED** [review]
ED for Kusama seems wrong according to on-chain info

**Comment by laboon** [comment]
_ED for Kusama seems wrong according to on-chain info_

Weird, I will look into this, thanks

File: docs/learn-accounts.md [hunk 3]
```diff
@@ -100,14 +100,14 @@ Hardware wallet integration from [Ledger](https://ledger.com) is coming soon.
...
 When you generate an account (address), you only generate a _key_ that lets you access it. The
-account does not exist yet on-chain. For that, it needs the existential deposit - 0.01 KSM / DOT.
-This deposit is paid by the account sending you tokens. The one "creating" you pays for your
-creation.
-
-Going below 0.01 causes an account to be _reaped_. Your account will be wiped from the blockchain's
-state to conserve space. You do not lose access to the reaped address - as long as you have your
-private key or recovery phrase, you can still use the address - but it needs a top-up of another
-existential deposit to be able to interact with the chain.
+account does not exist yet on-chain. For that, it needs the existential deposit: 0.001 KSM (on
```
Comment (joepetrowski) [inline docs/learn-accounts.md]:
```suggestion
account does not exist yet on-chain. For that, it needs the existential deposit: 0.001666666667 KSM (on
```

**Review by lsaether — APPROVED** [review]

**Pull request merged**